misc-utils/lsblk.c

```diff
@@ -84,6 +84,7 @@ enum {
 	COL_GROUP,
 	COL_HCTL,
 	COL_HOTPLUG,
+	COL_REMOVABLE,
```

Comment:
I guess "RM" is good enough in this case.

Reply:
> Same as USB flash drive:

I think the "removable" USB stuff is a mistake: https://lore.kernel.org/lkml/20220112140304.GA27446@lst.de/